pkg/util/version/const.go

```diff
@@ -89,7 +89,7 @@ func MdsdImage(acrDomain string) string {
 
 // MUOImage contains the location of the Managed Upgrade Operator container image
 func MUOImage(acrDomain string) string {
-	return acrDomain + "/managed-upgrade-operator:aro-b4"
+	return acrDomain + "/app-sre/managed-upgrade-operator:v0.1.952-44b631a"
```

Comment:
why did the `/app-sre/` change?

Reply:
Because the [upstream image builds in quay](https://quay.io/repository/app-sre/managed-upgrade-operator?tab=tags) are under that path, and our mirror code is later replicating it into ACR.

On the other hand, the previous aro-b4 image was a custom ARO build (needed for FIPS support at that time), which was specifically pushed to ACR, and not published under /app-sre repo.